Merge Conflict/Assets/Scripts/AudioManager/AudioManager.cs

```diff
@@ -0,0 +1,218 @@
+/**********************************************************************************************************************
+Name:          AudioManager
+Description:   Manages all sounds and background music that will be played.  
```

Comment:
Pls mention that the methods are to be used by other scripts and that the background music is handled by the AudioManager itself

Reply:
done